src/App.jsx

```diff
@@ -14,33 +15,116 @@ export const goodsFromServer = [
   'Garlic',
 ];
 
-export const App = () => (
-  <div className="section content">
-    <div className="buttons">
-      <button type="button" className="button is-info is-light">
-        Sort alphabetically
-      </button>
+export const App = () => {
+  const [goods, setGoods] = useState(goodsFromServer);
+  const [sortedAlph, setSortedAlph] = useState(false);
+  const [sortedByLength, setSortedByLength] = useState(false);
+  const [reverse, setReverse] = useState(false);
 
-      <button type="button" className="button is-success is-light">
-        Sort by length
-      </button>
+  const applySortingAndReverse = (array, sortType) => {
+    const updatedGoods = [...array];
 
-      <button type="button" className="button is-warning is-light">
-        Reverse
-      </button>
+    if (sortType === 'alphabetical') {
+      updatedGoods.sort((good1, good2) => good1.localeCompare(good2));
+    } else if (sortType === 'length') {
```

Comment:
- Use switch case instead
- 'alphabetical' and 'length' are magic values, create a constant